### PR TITLE
Added Codeption\Module\Apc module

### PR DESCRIPTION
**Pull request by sergeyklay**

### CODE REVIEW

File: src/Codeception/Module/Apc.php [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+     * ```
+     *
+     * @param $key
+     * @param $value
+     */
+    public function dontSeeInApc($key, $value = null)
+    {
+        if (null === $value) {
+            $this->assertFalse($this->exists($key), "The key '$key' exists in APC(u).");
+        } else {
```
Comment (DavertMik) [inline src/Codeception/Module/Apc.php]: you can have `return` one line above without `else`

File: src/Codeception/Module/Apc.php [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    /**
+     * Stores an item `$value` with `$key` on the APC(u).
+     *
+     * @param string $key
+     * @param mixed $value
+     * @param int $expiration
+     */
+    public function haveInApc($key, $value, $expiration = null)
+    {
+        $this->assertTrue($this->store($key, $value, $expiration));
```
Comment (DavertMik) [inline src/Codeception/Module/Apc.php]: there should not be assertions in `have*` methods. Just store it and return some key (if it is needed).
Also please provide an example in docblock

Comment (sergeyklay) [inline src/Codeception/Module/Apc.php]: I just repeating the Memcached logic
https://github.com/Codeception/Codeception/blob/2.2/src/Codeception/Module/Memcache.php#L183-L193

**Comment by sergeyklay** [comment]
@DavertMik Done

**Comment by DavertMik** [comment]
Thank you!

**Pull request merged**